### ISSUE TITLE
I think I fix a text rendering bug.

### ISSUE THREAD

**Issue by nojinadev** [TomRoush/PdfBox-Android#526]
in PageDrawer.java --------------------------------------------------------------------

private void drawGlyph2D(Glyph2D glyph2D, PDFont font, int code, Vector displacement,
        AffineTransform at) throws IOException
    {
        PDGraphicsState state = getGraphicsState();
        RenderingMode renderingMode = state.getTextState().getRenderingMode();

//        Path path = glyph2D.getPathForCharacterCode(code);
        Path path = new Path(glyph2D.getPathForCharacterCode(code)); -------> **fix like this. or better way.**

I think 

'path.transform(at.toMatrix());' transforms the path within the cache.

As a result, while the first character is rendered correctly, subsequent cached paths are transformed incorrectly, leading to issues.

**Comment by THausherr** [comment]
Please include the affected file.

**Comment by Tlanglord** [comment]
Yes, I also found this issue and submitted the pr, but it has not been merged yet
